app/build.gradle

```diff
@@ -53,15 +53,15 @@ android {
 }
 
 ext {
-    supportLibVersion = "26.0.1"
-    espressoVersion = "2.2.2"
-    androidTestVersion = "0.5"
-    hamcrestVersion = "1.3"
-    junitVersion = "4.12"
+    supportLibVersion = '27.0.2'
+    espressoVersion = '3.0.1'
+    androidTestVersion = '1.0.1'
+    hamcrestVersion = '1.3'
+    junitVersion = '4.12'
 }
 
 dependencies {
-    kapt "com.android.databinding:compiler:2.3.2"
+    kapt 'com.android.databinding:compiler:3.0.1'
```

Comment:
last thing (same comment for the change in top level build.gradle): all the dependency version names are extracted in the `ext` block - maybe you can do the same on this.

re: updating an external PR - does pushing to your branch not work?